try {
    console.log('a' - 20);
} catch (err1) {
    console.log(err1.message);
}



// console.log('1' * 2);